React/Views/RCTWebView.m

```diff
@@ -73,6 +73,11 @@ - (void)setURL:(NSURL *)URL
   [_webView loadRequest:[NSURLRequest requestWithURL:URL]];
 }
 
+- (void)setHTML:(NSString *)HTML
+{
+  [_webView loadHTMLString:HTML baseURL:nil];
```

Comment:
It supports dictionaries. You could do something like:

```js
<WebView html={string} baseURL={string} />
```

And in `WebView.js`

```js
<RCTWebView html={{html: this.props.html, basURL: this.props.baseURL}} />
```